src/azure-cli/azure/cli/command_modules/acr/_format.py

```diff
@@ -93,6 +93,55 @@ def agentpool_output_format(result):
     return _output_format(result, _agentpool_format_group)
 
 
+def connected_registry_output_format(result):
+    return _output_format(result, _connected_registry_format_group)
+
+
+def connected_registry_list_output_format(result):
+    family_tree = {}
+    for reg in result:
+        parent_id = _get_value(reg, 'parent', 'id')
+        parent_name = '' if parent_id.isspace() else parent_id.split('/connectedRegistries/')[1]
+        name = ' ' + _get_value(reg, 'name')
```

Comment:
what is `' '` prefix used for?

Reply:
That was to add an space before the '-' so when the users double click in the connected registry name it doesn't select as well all the '-'. And the table format doesn't print spaces so if they where level 0 it would be remove before printing. 
I have a todo to remove if the tree format is not shipped.